### ISSUE TITLE
Graph type and data format

### ISSUE THREAD

**Issue by andriusm** [vinted/more#1]
Currently the idea is to return the graph data for each item by attaching it to the item dto for mobile and using a helper for the web. We still need to decide

- type of graph that will be shown (bar chart, line chart, other..)
- data time window (e.g. 7 day rolling values)
- data format that frontend will receive it in

### Option 1

```
{
  // ... item dto ...
  
  "item_impressions": [
    {
      "date": "2015-04-03",
      "impressions": 5
    },
    {
      "date": "2015-04-04",
      "impressions": 9
    }
  ]
}
```

### Option 2

```
{
  // ... item dto ...

  "item_impressions": [ 5, 9, 3, 1, 7 ]
}
```

@vinted/more

**Comment by mjurkus** [comment]
For current MVP option 2 is enough.

If we're planning to display bigger chart in separate screen on mobile - we might need Option 1.

That solely depends on design.

cc @jonasgri

**Comment by GedmantasR** [comment]
> For current MVP option 2 is enough.

Yep, it is. Though looking into future we will definitely will need more data, simple example:

```json
 {
  "item_stats": [
    {
      "date": "2015-04-03",
      "impressions": 5,
      "views": 4
    },
    {
      "date": "2015-04-04",
      "impressions": 9,
      "views": 5
    }
  ]
}
```

**Comment by andriusm** [comment]
I know that option 2 is more concise, but I'm guessing that frontends will have to recreate a structure similar to option 1 in order to render the chart. From backend perspective the effort to provide the data in one format or the other will be pretty much the same. Hence I would propose option 1 or any other data format that we don't intend to change, so that it wouldn't cause issues for different mobile app versions.
